website/src/pages/showcase/index.js

```diff
@@ -43,9 +43,9 @@ const Content = (props) => {
   const showcases = siteConfig.customFields.showcases.map((user) => (
     <div className="col col--2 item" key={user.infoLink}>
       <UserCard
-          image={'https://cdn.jsdelivr.net/gh/apache/apisix-website@master/website/static/img/' + user.image}
-          caption={user.caption}
-          infoLink={user.infoLink}
+        image={'../../../static/img/' + user.image}
```

Comment:
Here you should continue to use the CDN to speed up the link

Reply:
@guoqqqi Hello,I have improved my documents.PTAL.😄